can_tools/bootstrap_data/locations.csv

```diff
@@ -238,7 +238,7 @@ iso1:us#iso2:us-mt#fips:30015,30015,30,30,Chouteau,3972.6726,47.886833,-110.4362
 iso1:us#iso2:us-ne#fips:31017,31017,31,31,Brown,1221.3971,42.359562,-99.92392,Brown County,county
 iso1:us#iso2:us-tn#fips:47013,47013,47,47,Campbell,480.2059,36.401592,-84.15925,Campbell County,county
 iso1:us#iso2:us-nc#fips:37069,37069,37,37,Franklin,491.8171,36.08824,-78.28309,Franklin County,county
-iso1:us#iso2:us-md#fips:24510,24510,24,24,Baltimore city,80.94945,39.300034000000004,-76.61046999999999,Baltimore city,county
+iso1:us#iso2:us-md#fips:24510,24510,24,24,Baltimore City,80.94945,39.300034000000004,-76.61046999999999,Baltimore City,county
```

Comment:
This looks like a good change. Are there other scrapers that may have been using the non-titlecase version?

Reply:
I'll answer my own question here... tests pass for all the Maryland scrapers. We should be gtg